lib/3scale_toolbox/commands/activedocs_command/list_command.rb

```diff
@@ -42,8 +43,32 @@ def print_header(fields_to_show)
             puts fields_to_show.map { |e| e.upcase }.join("\t")
           end
 
+          def service_ref_filter
+            options[:'service-ref']
+          end
+
+          def filters
+            res = []
+            if !service_ref_filter.nil?
+              res << AttributeFilters::ServiceIDFilterFromServiceRef.new(remote, service_ref_filter, "service_id")
+            end
+            res
+          end
+
+          def find_service(service_ref)
```

Comment:
This private method is being used?

Reply:
If we are going to use the approach of finding the service in the Filter this does not belong here indeed anymore and it's not used. I'm going to remove it :+1: